src/yachtzee.ts

```diff
@@ -331,13 +334,14 @@ export function yachtzeeChainDiet(simOnly?: boolean): boolean {
   // 4) Find meat and famwt buff that makes sense for a 2k base drop
   // 5) Plant underwater friar's plant if possible
 
-  const havePYECCharge = have($item`Platinum Yendorian Express Card`) && !get(`expressCardUsed`);
+  const havePYECCharge =
+    have($item`Platinum Yendorian Express Card`) && !get(`expressCardUsed`, false);
```

Comment:
You shouldn't have to specify a default value for `get` with native mafia preferences. I'm very curious what's causing your weird get issues.

Reply:
![libramGetWeirdness](https://user-images.githubusercontent.com/98746573/171626064-85a506f7-2947-406b-a160-e0d8e7684a2c.png)
Managed to get a screengrab of the `get` issue with _customPrefs, just in case you have an inkling of what might be going on